client/src/components/ParticlesBackground.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Particles from 'react-particles-js';
+import colors from '../utils/colorsUtils';
+import useWindowDimensions from '../utils/hooks';
+
+function ParticlesBackground() {
+  const { height, width } = useWindowDimensions();
+  const particleNumbers =
+    width < 500 ? (height + width) / 25 : (height + width) / 15;
+  return (
+    <>
+      <Particles
+        style={{
```

Comment:
Instead of inline-styles, better make a style component which will set `className`.

```
const BackgroundParticles = styled(Particles)`
  background: `linear-gradient(200deg, ${colors.primaryDark}, ${colors.backgroundSecondary})`,
  zIndex: '-1',
  width: '100%',
  height: '100%',
  position: 'fixed',
  top: '0',
  left: '0',
`;
```

Reply:
I tried this at first, but it didn't work. I think this Particles component needs the styles as props...

Reply:
Got it, you have to set `canvasClassName` to style the `canvas`. 

One solution could be this:

```
const BackgroundParticles = styled(Particles)`
  canvas {
    background: linear-gradient(200deg, ${colors.primaryDark}, ${colors.backgroundSecondary});
    zIndex: -1;
    width: 100%;
    height: 100%;
    position: fixed;
    top: 0;
    left: 0;
  }
`;
```

Or 
```
const BackgroundParticles = styled(props => <Particles canvasClassName={props.className} /> )`
    background: linear-gradient(200deg, ${colors.primaryDark}, ${colors.backgroundSecondary});
    zIndex: -1;
    width: 100%;
    height: 100%;
    position: fixed;
    top: 0;
    left: 0;

`;
```